README.md

```diff
@@ -63,13 +65,17 @@ In the Arguments section, add five (or more) Arguments Passed On Launch to the b
   ```
   --base-path="/Users/YourName/Desktop/YourProjectFolder"
   ```
-- The path within your project to your asset catalog, without a preceding slash (add multiple times if you have multiple asset catalogs:  
+- The path within your project to your asset catalog, without a preceding slash (add multiple times if you have multiple asset catalogs):  
   ```
   --asset-catalog="Resources/Images.xcassets"
   ```  
   ```
   --asset-catalog="Resources/Media.xcassets"
   ```
+- The prefix for the category method names. If not specified, the default is `ac`. Must be at least 2 characters, and conform to valid method naming rules:
```

Comment:
One thing you mention is possible in your PR comment but isn't quite clear in this documentation: What happens if multiple asset catalogs are specified? 

Do you need to add multiple prefixes for the method name? Is the first method name prefix passed in used for every catalog? How do you specify which prefix is for which catalog?